src/react-native/07-debugging-devtools/debugging-devtools.md

```diff
@@ -16,13 +16,8 @@ In this section, you will:
 
 We will be setting up React DevTools to help us inspect the UI:
 
-**React DevTools**
-
 <img alt="Screenshot of React DevTools inspecting a component tree. It shows a structured hierarchy of all the components in the view. There is also a Profiler tab." src="../../static/img/react-native/07-debugging-devtools/dev-tool-connected.png" style="max-width: 100%;"/>
 
-### Concept TODO
-TODO
-
```

Comment:
We should still have a concept here to explain what React DevTools is and what it enables you to do. It will be short but it’s worth having a paragraph or two to explain.

I’d also put the app’s JSX here so we can see what the component tree looks like in JSX, and how that maps to what we see in DevTools.